delphi-frontend/src/test/java/au/com/integradev/delphi/type/intrinsic/IntrinsicReturnTypeTest.java

```diff
@@ -35,83 +35,113 @@
 import org.sonar.plugins.communitydelphi.api.type.TypeFactory;
 
 class IntrinsicReturnTypeTest {
-  private static final TypeFactory TYPE_FACTORY =
+  private static final TypeFactory TYPE_FACTORY_32_RIO =
+      new TypeFactoryImpl(Toolchain.DCC32, CompilerVersion.fromVersionNumber("35.0"));
+  private static final TypeFactory TYPE_FACTORY_32_ATHENS =
+      new TypeFactoryImpl(Toolchain.DCC32, CompilerVersion.fromVersionNumber("36.0"));
+  private static final TypeFactory TYPE_FACTORY_64_RIO =
       new TypeFactoryImpl(Toolchain.DCC64, CompilerVersion.fromVersionNumber("35.0"));
```

Comment:
You've jumped an extra version backwards for these variable names 😄

```suggestion
  private static final TypeFactory TYPE_FACTORY_32_ALEXANDRIA =
      new TypeFactoryImpl(Toolchain.DCC32, CompilerVersion.fromVersionNumber("35.0"));
  private static final TypeFactory TYPE_FACTORY_32_ATHENS =
      new TypeFactoryImpl(Toolchain.DCC32, CompilerVersion.fromVersionNumber("36.0"));
  private static final TypeFactory TYPE_FACTORY_64_ALEXANDRIA =
      new TypeFactoryImpl(Toolchain.DCC64, CompilerVersion.fromVersionNumber("35.0"));
```

Reply:
Oops. 👀